.github/workflows/release.yml

```
@@ -41,16 +41,21 @@ jobs:
-          footer: |
-            View Release: ${{ github.event.release.html_url }}
+          title: New Update Available
+          description: |
+            ## :loudspeaker: Tailscale Plugin Update : ${{ github.event.release.name }}
+          fields: |
+            [
+              {"name": ":wrench: Changes", "value": ${{ toJson(github.event.release.body) }} },
+              {"name": "Channel", "value": ${{ toJson(github.event.release.prerelease && ':test_tube: Preview' || 'Release') }}, "inline": true},
+              {"name": ":label: Tag", "value": ${{ toJson(github.event.release.tag_name) }}, "inline": true},
+              {"name": ":link: Release Link", "value": ${{ toJson(github.event.release.html_url) }} }
+            ]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)